packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -199,246 +214,224 @@ export const CreateOAuthModal: React.FunctionComponent<
+        onChange={e => setSectorUri(e.target.value)}
+        placeholder="https://"
+        type="text"
+        value={sectorUri}
+        id="sectorURI"
+      />
+    </StyledFormControl>
+  )
 
-                            {redirectUris.length - 1 === idx && (
-                              <Button
-                                onClick={handleRedirectUriAdd}
-                                disabled={singleUri.uri.length === 0}
-                              >
-                                Add URI
-                              </Button>
-                            )}
-                          </div>
-                        ))}
-                      </Form.Group>
-                    </Col>
-                    <Col lg={6} md={6} sm={12} xs={12}>
-                      <Form.Label>Sector Identifier URI</Form.Label>
-                      <Form.Control
-                        onChange={e => setSectorUri(e.target.value)}
-                        placeholder="https://"
-                        type="text"
-                      />
-                    </Col>
-                  </>
-                )}
-              </Row>
-              <Row>
-                <Col lg={6} md={6} sm={12} xs={12}>
-                  <Form.Label>Link to Privacy Policy</Form.Label>
-                  <Form.Control
-                    onChange={e => setPolicyUri(e.target.value)}
-                    placeholder="https://"
-                    type="text"
-                    value={policyUri}
-                  />
-                </Col>
-                <Col lg={6} md={6} sm={12} xs={12}>
-                  <Form.Label>Links to Terms of Service</Form.Label>
-                  <Form.Control
-                    onChange={e => setTosUri(e.target.value)}
-                    placeholder="https://"
-                    type="text"
-                    value={tosUri}
-                  />
-                </Col>
-              </Row>
-              {isEdit && (
-                <div className="danger">
-                  <Typography
-                    style={{ marginTop: '8px' }}
-                    color="error"
-                    variant="headline3"
-                  >
-                    DANGER ZONE
-                  </Typography>
-                  <Typography variant="smallText1">
-                    Editing the following information will render your client
-                    invalid and will require you to create it again and resubmit
-                    verification if needed.
-                  </Typography>
-                  <Row>
-                    <Col lg={6} md={6} sm={12} xs={12}>
-                      <Form.Label htmlFor="redirect-uri-0">
-                        Redirect URI(s)
-                      </Form.Label>
-                      <Tooltip title={uriHelpMessage} placement="top">
-                        <HelpOutlineTwoTone className={`HelpButton`} />
-                      </Tooltip>
+  const redirectURIInputs = (
```

Comment:
Consolidate definitions for Redirect URI inputs to one location for creating vs editing a client